docs/getting-started/installing-and-running.mdx

```diff
@@ -66,15 +68,21 @@ Then, you can run Conduit by running:
 docker run -it -p 8080:8080 conduit.docker.scarf.sh/conduitio/conduit
 ```
 
-The command aboves runs Conduit with no pipelines. To run Conduit with a custom pipeline file you can mount it as a volume into the container:
+The command aboves runs Conduit with no pipelines. To run Conduit with a custom
+pipeline file you can mount it as a volume into the container:
+
 ```shell
 docker run -it -p 8080:8080 \
   -v /path/to/pipeline.yaml:/app/pipelines/pipeline.yaml \
   conduit.docker.scarf.sh/conduitio/conduit
 ```
 
 :::tip
-If you're using Docker, check out [Watchtower](https://containrrr.dev/watchtower/). After we push an update, Watchtower will keep your Conduit container updated by gracefully shutting down your existing container and restarting it with the same options used when initially deployed.
+If you're using Docker, check
+out [Watchtower](https://containrrr.dev/watchtower/). After we push an update,
+Watchtower will keep your Conduit container updated by gracefully shutting down
+your existing container and restarting it with the same options used when
+initially deployed.
 :::
 
   </TabItem>
```

Comment:
2 lines below we mention you need to have "Go 1.22" or higher. I'd remove the Go version, as we constantly update it and this will get outdated in a few months. Let's just say "latest version of Go".

Oh and while we're at it, we could remove that part of building Conduit without the UI, that will be the default soon anyway.